frontend/src/services/effects.ts

```diff
@@ -246,6 +246,18 @@ export function withEffects(store: Store) {
     }
     )
 
+  /**
+   * Rebuild the geojson whenever the adequacies change.
+   */
+  store
+  .on('adequacies')
+  .subscribe(adequacies => {
+    if (adequacies){
```

Comment:
Use a filter instead of an `if`:

```ts
store
  .on('adequacies')
  .filter(Boolean)
  .subscribe(..)
```

Reply:
Ooo thanks! Guess who has two thumbs and no idea about RXJS...